clustered/integration-test/src/test/java/org/ehcache/clustered/management/ClusteredStatisticsRatioTest.java

```diff
@@ -15,31 +15,31 @@
  */
 package org.ehcache.clustered.management;
 
-import static org.ehcache.clustered.management.AbstractClusteringManagementTest.cacheManager;
-import static org.ehcache.clustered.management.AbstractClusteringManagementTest.sendManagementCallToCollectStats;
-import static org.hamcrest.CoreMatchers.is;
-import static org.junit.Assert.assertThat;
-
-import java.util.List;
 import org.ehcache.Cache;
-import org.junit.Assert;
 import org.junit.Test;
-import org.terracotta.management.model.call.ContextualReturn;
 import org.terracotta.management.model.stats.ContextualStatistics;
 import org.terracotta.management.model.stats.Sample;
 import org.terracotta.management.model.stats.history.RatioHistory;
 
-public class ClusteredStatisticsRatioTest extends AbstractClusteringManagementTest {
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.DoubleStream;
 
-  private static final double CACHE_HIT_RATIO = .5d;
-  private static final double CLUSTERED_HIT_RATIO = .5d;
-  private static final double CACHE_MISS_RATIO = .5d;
-  private static final double CLUSTERED_MISS_RATIO = .5d;
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.Matchers.array;
+import static org.junit.Assert.assertThat;
+
+public class ClusteredStatisticsRatioTest extends AbstractClusteringManagementTest {
 
   @Test
   public void ratioTest() throws Exception {
-    ContextualReturn<?> contextualReturn = sendManagementCallToCollectStats("Cache:HitRatio","Clustered:HitRatio","Cache:MissRatio","Clustered:MissRatio");
-    assertThat(contextualReturn.hasExecuted(), is(true));
+    String[] statNames = {"Cache:HitRatio", "Clustered:HitRatio", "Cache:MissRatio", "Clustered:MissRatio"};
+    sendManagementCallToCollectStats(statNames);
+
+    // See https://goo.gl/hqjidi for explanations why we need to wait
+    Thread.sleep(3000);
```

Comment:
Because there is the clustered layer... This mangement call just says: start collecting stats. And the collector runs at 75% of the time of the TTD. Ss we need to take that in consideration too.